examples/imageLoad/image.yaml

```diff
@@ -0,0 +1 @@
+URL: "http://localhost:8080/httpd.tar"
```

Comment:
this isn't used, right?

Reply:
oh i see, it is the targetPath for example Image

Reply:
I'm thinking this URL should be included in the Image type. It will be a type that is independent of Git, like the ConfigTarget is.